ldes-server-infra-mongo/mongock-changeset-3/README.md

```diff
@@ -0,0 +1,21 @@
+# Mongock Changeset 3
+
+The following configuration is needed to apply this changeset.
+
+This changeset relies on the ldes configuration to assign members and fragments to a collection and impose an order on the members. When more than one collection is configured,
+the migration does not know which version or timestamp to use. In this case it will use `UNDEFINED_COLLECTION` as collectionName. The reason why it uses this value instead of `null`, is because the collectionName is added as prefix to ids and properties. Adding `null` might lead to unexpected behaviour. Using the value `UNDEFINED_COLLECTION`, it will be easier to adapt or rollback if needed.
```

Comment:
why do you refer to 'version or timestamp' ? Isn't that from a previous changeset?

Reply:
Haha, you got me! :-) Busted